library/swift/src/filters/RequestFilterCallbacksImpl.swift

```diff
@@ -0,0 +1,18 @@
+@_implementationOnly import EnvoyEngine
+import Foundation
+
+/// Envoy implementation of RequestFilterCallbacks
+final class RequestFilterCallbacksImpl: NSObject {
+  private let callbacks: EnvoyHTTPFilterCallbacks
+
+  init(callbacks: EnvoyHTTPFilterCallbacks) {
+    self.callbacks = callbacks
+    super.init()
+  }
+}
+
+extension RequestFilterCallbacksImpl: RequestFilterCallbacks {
+  func resumeRequest() {
+    callbacks.resumeIteration()
```

Comment:
```suggestion
    self.callbacks.resumeIteration()
```

Reply:
Cool; for same reason every language has a different convention around this. :p

Reply:
yea the bias I've kept in this repo (same at Lyft) is to explicitly use `self.` to make the source clearer (it's required within closures as well)